### DISCUSSION TITLE
Twitter (20K) Giveaway Winners

### DISCUSSION THREAD

**Discussion by ehsandeep** [projectdiscovery/nuclei#2595] [Show and tell]
### Giveaway announcement
  - https://twitter.com/pdnuclei/status/1566754415247106049
  
### Giveaway duration
  - **September 05**, 2022 - **September 11**, 2022
  
### Total Participant
 - 78
 
### Random selection

https://user-images.githubusercontent.com/8293321/189670660-fa5c0b0d-353a-4fbd-bc14-3be936f09e4c.mp4

<details>
<summary><b>Winners list (40): </b></summary>

1. @akincibor
2. @Abid-Ahmad
3. @jwick1979
4. @Jaypanchal1401
5. @themarkib
6. @johnk3r
7. @zeeshan1991k
8. @Linuxinet
9. @ajmalabubakkr
10. @Saket-taneja
11. @MalwareJuice
12. @othaime-en
13. @Ajm41
14. @yosif-qasim
15. @brutexploiter
16. @48h153k
17. @pikpikcu
18. @casmpy
19. @noraj
20. @HackerAndya
21. @0xhunster
22. @arpitkarnak
23. @NickBot09
24. @Suman-Neupane
25. @anuragdadhich78
26. @bhuwanpatidar
27. @AbhinavCSY
28. @hackergod00001
29. @kubota
30. @jini0x
31. @brenocss
32. @bakliwalp
33. @nextep
34. @RAVE-V
35. @aashisham
36. @0xjashim
37. @SagarDhoot
38. @ramsayotaku
39. @nigamelastic
40. @theoricien

</details>

### How to receive vouchers?

To receive [PentesterLab](https://twitter.com/PentesterLab) vouchers:

- Join the discord server - https://discord.gg/projectdiscovery
- Direct message to [forgedhallpass](https://twitter.com/forgedhallpass) (`forgedhallpass#3419`) on Twitter or Discord
- Verify the identity and receive the vouchers.

**Comment by MalwareJuice** [comment]
Thank you :)

**Reply by Ajm41** [reply]
Thank you

**Comment by daffainfo** [comment]
Ahh too bad, I didn't win. But I am waiting for the big update on nuclei :)

**Reply by forgedhallpass** [reply]
Next time! 😉

**Comment by brutexploiter** [comment]
Thank you :⁠-⁠)

**Comment by 0xjashim** [comment]
thank you

**Comment by nigamelastic** [comment]
Thanks somehow I missed it earlier I hope I get it now :)

**Reply by ehsandeep** [reply]
@nigamelastic please dm on discord to @forgedhallpass if you missed collecting your voucher.

**Reply by nigamelastic** [reply]
Thanks @ehsandeep I already did

**Reply by nigamelastic** [reply]
hopefully he will see it soon and then get back to me :)

**Comment by Jaypanchal1401** [comment]
thank you sir
